### ISSUE TITLE
show voice duration before playing the audio

### ISSUE THREAD

**Issue by ghost** [hyochan/react-native-audio-recorder-player#248]
I'm going to show voice duration to user before start playing the voice? what should i suppose to do?

**Comment by brycechampaign** [comment]
Yeah, I would like to know if this is possible as well.

**Comment by Sargnec** [comment]
Set current position in 
` audioRecorderPlayer.addRecordBackListener((e) => {
            setRecordTime(e.current_position)
            return;
        });`
And save it in stopRecord or something?

**Comment by hyochan** [comment]
I think this is already covered in our `Example` project.

**Comment by milap-encora** [comment]
@hyochan unable to find it in the example we get duration `onStartPlay`  we need to get duration before starting the Audio.
Let me know how can we get the duration before starting the player.
Thanks in advance.
